dist/FormView.js

```diff
@@ -42,7 +42,7 @@
 
       //Attach Events to preexisting elements if we don't have a template
       if (!this.template) this.runInitializers();
-      this.on('item:rendered',this.runInitializers, this);
```

Comment:
Any progress thoughts on getting this merged so projects with newer versions of marionette can leverage this package?